client/src/lib.rs

```diff
@@ -231,29 +238,12 @@ impl<C: Deref<Target = impl Signer> + Clone> Program<C> {
         self.program_id
     }
 
-    pub fn rpc(&self) -> RpcClient {
-        RpcClient::new_with_commitment(
-            self.cfg.cluster.url().to_string(),
-            self.cfg.options.unwrap_or_default(),
-        )
-    }
-
-    pub fn async_rpc(&self) -> AsyncRpcClient {
-        AsyncRpcClient::new_with_commitment(
-            self.cfg.cluster.url().to_string(),
-            self.cfg.options.unwrap_or_default(),
-        )
-    }
```

Comment:
I think having only the `rpc` method is better, but since this is a breaking change, could you note the removal of this public method in the [CHANGELOG](https://github.com/coral-xyz/anchor/blob/117717468f44799ceb4b98f5da65c6d7a1b7d8c6/CHANGELOG.md#breaking)?